webpack/components/OpenscapRemediationWizard/steps/ReviewHosts.js

```diff
@@ -0,0 +1,198 @@
+import React, { useContext, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Spinner,
+  Toolbar,
+  ToolbarContent,
+  ToolbarGroup,
+  ToolbarItem,
+} from '@patternfly/react-core';
+import { Td } from '@patternfly/react-table';
+
+import { foremanUrl, noop } from 'foremanReact/common/helpers';
+import { translate as __ } from 'foremanReact/common/I18n';
+import SearchBar from 'foremanReact/components/SearchBar';
+import SelectAllCheckbox from 'foremanReact/components/PF4/TableIndexPage/Table/SelectAllCheckbox';
+import { Table } from 'foremanReact/components/PF4/TableIndexPage/Table/Table';
+import { useBulkSelect } from 'foremanReact/components/PF4/TableIndexPage/Table/TableHooks';
+import { getPageStats } from 'foremanReact/components/PF4/TableIndexPage/Table/helpers';
+import { getControllerSearchProps, STATUS } from 'foremanReact/constants';
+import { useAPI } from 'foremanReact/common/hooks/API/APIHooks';
+
+import OpenscapRemediationWizardContext from '../OpenscapRemediationWizardContext';
+import WizardHeader from '../WizardHeader';
+import {
+  HOSTS_API_PATH,
+  HOSTS_API_REQUEST_KEY,
+  FAIL_RULE_SEARCH,
+} from '../constants';
+
+const ReviewHosts = () => {
+  const { source, hostId, setHostIdsParam } = useContext(
+    OpenscapRemediationWizardContext
+  );
+
+  const defaultSearch = `${FAIL_RULE_SEARCH} = ${source}`;
+  const defaultParams = {
+    search: defaultSearch,
+  };
+
+  const [params, setParams] = useState(defaultParams);
+
+  const searchProps = getControllerSearchProps('hosts');
+  searchProps.autocomplete.searchQuery = defaultSearch;
+  searchProps.autocomplete.url = '../../hosts/auto_complete_search'; // TODO: find a way to not use relative path
+  searchProps.disabled = true; // This is to force hosts to be searched by currently remediated rule only
```

Comment:
If the search is always disabled, why is there an "onSearch" function, or a search bar in general?

Reply:
Initially I wanted to allow the search, but after more thinking I've decided to just disable it and show users why these hosts are shown exactly. Probably can be done via simple note entry.